### PR TITLE
Feat: Added questions to test directory

### PR DESCRIPTION
**Pull request by Moibrahi7**

### CODE REVIEW

File: lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/MohamedIbrahimQuiz.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  public List<QuizQuestion> makeQuizQuestions() {
+    return List.of(makeQuestion0(), makeQuestion1());
+  }
+
+  private QuizQuestion makeQuestion0() {
+    return new MultipleChoiceQuizQuestion(
+        0,
+        "What are essential parts to building a computer? (without the inclution of a case)",
```
Comment (anthonydmays) [inline lesson_03/quiz/lesson_03_quiz/src/main/java/com/codedifferently/lesson3/quizzes/MohamedIbrahimQuiz.java:25]: Spelling

File: lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml [hunk 2]
```diff
@@ -6,3 +6,7 @@ quiz:
     anthonymays:
      - $2y$10$8eHSzy3aCu4Ry3LzO9nWCeGpofSxsNVbnF.wCfn3ZADwQ6MEtN/KK
      - $2y$10$55EXRjF26JIgebtoH800ZOJecfefvMgHicuxf/rwTENuxiUaFQcNe
+    moibrahi7:
+        - $2y$10$qMRtpEAfuW9e5WyZv5qzxuTj3JDO4xuOrNwm1FZuQ9uGcm3NHZMC.
```
Comment (anthonydmays) [inline lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml:10]: Match existing formatting.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Comment by Moibrahi7** [comment]
Bug: code produces error for lesson_02 for an unknown reason

File: lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml [hunk 3]
```diff
@@ -6,3 +6,7 @@ quiz:
     anthonymays:
      - $2y$10$8eHSzy3aCu4Ry3LzO9nWCeGpofSxsNVbnF.wCfn3ZADwQ6MEtN/KK
      - $2y$10$55EXRjF26JIgebtoH800ZOJecfefvMgHicuxf/rwTENuxiUaFQcNe
+    moibrahi7:
+      - $2y$10$qMRtpEAfuW9e5WyZv5qzxuTj3JDO4xuOrNwm1FZuQ9uGcm3NHZMC.
```
Comment (anthonydmays) [inline lesson_03/quiz/lesson_03_quiz/src/test/resources/application.yml:10]: Indentation is off by a space, please fix.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**